### ISSUE TITLE
Warnings when installing package

### ISSUE THREAD

**Issue by paymog** [FleekHQ/fleek-storage-js#28]
I get the following warnings when installing the package:

```
 ❮❮❮ yarn add @fleekhq/fleek-storage-js

yarn add v1.23.0-20210726.1745
[1/4] 🔍  Resolving packages...
warning @fleekhq/fleek-storage-js > text-encoding@0.7.0: no longer maintained
warning @fleekhq/fleek-storage-js > aws-sdk > querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
warning @fleekhq/fleek-storage-js > aws-sdk > url > querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
warning @fleekhq/fleek-storage-js > aws-sdk > uuid@3.3.2: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
```

Are there plans to resolve these warnings?

**Comment by SamueleA** [comment]
You are right! There is indeed maintenance that needs to be done on the Fleek Storage Js package.

We will definately address it. Thank you for pointing it out!

**Comment by SamueleA** [comment]
Error message have been fixed in 1.0.18.
The only error message that have not been removed are those generated by aws-sdk. Even after upgrading to the most recent version the warnings still show up so nothing can be done about those.

**Comment by dfelix** [comment]
2.1074.0 is returning same warnings...

warning aws-sdk > querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
warning aws-sdk > url > querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
warning aws-sdk > uuid@3.3.2: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
